packages/contracts-versions/README.md

```diff
@@ -16,15 +16,35 @@ yarn add @aragon/osx-versions
 
 ## Usage
 
+Import Active contracts:
+
+```javascript
+// import active contracts from a specific version
+import {v1_2_0_active_contracts} from '@aragon/osx-versions';
+
+const mumbaiActiveContracts = v1_2_0_active_contracts.mumbai;
+```
+
+Import a specific contract source code from a specific version:
+
+```javascript
+
+// SPDX-License-Identifier: AGPL-3.0-or-later
+
+pragma solidity 0.8.17;
+
+// import legacy contracts from a specific version
+import {DAO} from '@aragon/osx-versions/versions/v1_0_1/contracts/core/dao/DAO.sol';
```

Comment:
Are we going to publish this to NPM?

Reply:
i guess we agreed not to, unless there is demand for it, for now it is for internal use only